HHS/Head-Start-TTADP
# [TTAHUB-1300] Fix display of grants for multi grant goals on the RTR

## Description of change
This PR fixes a bug where the wrong API field was accessed when editing or viewing an existing goal with multiple grants on the RTR.( All grants are returned but only one was populating the select or read-only field)

## How to test
Create a multi grant goal on the RTR. Save, submit, and edit. You will see that the proper number of grants is displayed throughout. Leave the RTR. Return to that goal or another multi-grant goal. You will still see the proper number of grants, whether the goal is editable or read-only.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1300


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
